cclib/parser/orcaparser.py

```diff
@@ -785,40 +809,37 @@ def splitter(line):
             thermal_vibrational_correction = float(next(inputfile).split()[4])
             thermal_rotional_correction = float(next(inputfile).split()[4])
             thermal_translational_correction = float(next(inputfile).split()[4])
-            next(inputfile)
+            self.skip_lines(inputfile, ['dashes'])
             total_thermal_energy = float(next(inputfile).split()[3])
 
             # Enthalpy
-            line = next(inputfile)
             while line[:17] != 'Total free energy':
                 line = next(inputfile)
             thermal_enthalpy_correction = float(next(inputfile).split()[4])
             next(inputfile)
-
-            if self.natom > 1:
-                self.enthalpy = float(next(inputfile).split()[3])
-            else:
-                self.enthalpy = self.electronic_energy + thermal_translational_correction
+            self.enthalpy = float(next(inputfile).split()[3])
```

Comment:
This enthalpy would be wrong for single atoms due to spurious translational frequencies ORCA doesn't remove in this case ([as discussed here](https://github.com/cclib/cclib/pull/817#discussion_r395715175)).

Reply:
Gotcha - added a comment about this then, in the code and in `regressions.py`. Thanks for point this out!